aws/logs_monitoring/installation_alternatives.md

```diff
@@ -33,6 +33,36 @@ resource "aws_cloudformation_stack" "datadog-forwarder" {
   template_url = "https://datadog-cloudformation-template.s3.amazonaws.com/aws/forwarder/latest.yaml"
   
 }
+
+# Example Cloudwatch Logs subscription for AWS Batch
+
+variable "aws_region" {
+  type        = string
+  description = "AWS Region"
+  default     = "us-east-1"
+}
+
+variable "account_id" {
+  type = string
+  description = "AWS Account Id"
+}
+
+resource "aws_cloudwatch_log_subscription_filter" "aws_batch" {
```

Comment:
We have this documented in this section https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#manually-setup-triggers

Reply:
I'm very happy to see the docs now, thank you, I missed it somehow.  Nice work!  Feel free to close the pull request :-)